modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/VerifyBackupPartitionsTask.java

```diff
@@ -330,9 +332,9 @@ else if (e.getCause() instanceof IgniteException)
                 long updateCntrAfter = part.updateCounter();
 
                 if (updateCntrBefore != updateCntrAfter) {
-                    throw new IgniteException("Cluster is not idle: update counter of partition [grpId=" +
-                        grpCtx.groupId() + ", partId=" + part.id() + "] changed during hash calculation [before=" +
-                        updateCntrBefore + ", after=" + updateCntrAfter + "]");
+                    throw new GridNotIdleException(GRID_NOT_IDLE_MSG + "[grpName=" + grpCtx.cacheOrGroupName() +
```

Comment:
The check above will not detect out-of-order updates. We should compare whole counter.

Reply:
VerifyBackupPartitionsTaskV2 now used, VerifyBackupPartitionsTask looks like outdated stuff and need to be removed, i fixed it there:VerifyBackupPartitionsTaskV2.VerifyBackupPartitionsJobV2#calculatePartitionHash